backend/projects/models.py

```diff
@@ -57,6 +58,12 @@ class Meta:
         unique_together = ("project", "user")
 
 
+class MetricsPreferences(BaseModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    metrics = ArrayField(models.CharField(max_length=100))
+
+
```

Comment:
Ah yes also you should add a `Meta` with `unique_together` to ensure the unicity of the `user` and `project` couple https://docs.djangoproject.com/en/3.0/ref/models/options/#unique-together